lib/src/bazelify/arguments.dart

```diff
@@ -9,13 +9,114 @@ import 'package:args/args.dart';
 import 'package:path/path.dart' as p;
 import 'package:which/which.dart';
 
-/// Arguments when running `bazelify`, which adds Bazel support on top of pub.
+/// Shared arguments between all bazelify commands.
 class BazelifyArguments {
-  static final ArgParser _argParser = new ArgParser()
-    ..addOption(
-      'bazel',
-      help: 'A path to the "bazel" executable. Defauls to your PATH.',
-    )
+  /// The parser for shared arguments
```

Comment:
[nit] what does this tell us that the signature + context don't already tell us?